arch/arm/src/armv7-m/arm_assert.c

```diff
@@ -137,54 +137,162 @@ static inline void up_registerdump(FAR volatile uint32_t *regs)
  * Name: up_taskdump
  ****************************************************************************/
 
-#if defined(CONFIG_STACK_COLORATION) || defined(CONFIG_SCHED_BACKTRACE)
-static void up_taskdump(FAR struct tcb_s *tcb, FAR void *arg)
+static void up_dump_task(FAR struct tcb_s *tcb, FAR void *arg)
```

Comment:
let's update other arm variant and arch too.

Reply:
on next phase, coming soon

Reply:
Ok.